src/panels/AddSecretPanel/Fields/Fields.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { screen } from "@testing-library/react";
+import { Formik } from "formik";
+
+import { renderComponent } from "testing/utils";
+
+import Fields from "./Fields";
+
+describe("Fields", () => {
+  it("renders", async () => {
```

Comment:
This is just a placeholder test for when the functionality is added in https://warthogs.atlassian.net/browse/WD-8546.